lib/bpk-appearance/src/BpkAppearance-test.js

```diff
@@ -15,98 +15,82 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { eventEmitter as mockedEmitter } from 'react-native-dark-mode';
 
-import subject from './BpkAppearance';
-
-jest.mock('react-native-dark-mode', () => {
-  const eventEmitter = (() => {
-    let registeredCallback = () => {};
-    return {
-      on: (event, callback) => {
-        registeredCallback = callback;
-      },
-      trigger: (newMode) => registeredCallback(newMode),
-    };
-  })();
-  return {
-    initialMode: 'light',
-    eventEmitter,
-  };
-});
+import appearance from './BpkAppearance';
 
 describe('BpkAppearance', () => {
-  const initialAppearance = subject.get();
+  const initialAppearance = appearance.get();
   afterEach(() => {
-    subject.set(initialAppearance);
+    appearance.set(initialAppearance);
   });
 
   it('correctly returns initial appearance', () => {
-    expect(subject.get()).toEqual({ colorScheme: 'light' });
+    expect(appearance.get()).toEqual({ colorScheme: 'light' });
   });
 
   it('correctly updates the appearance', () => {
-    expect(subject.get()).toEqual({ colorScheme: 'light' });
+    expect(appearance.get()).toEqual({ colorScheme: 'light' });
```

Comment:
this test seems the same as the above?

Reply:
You are right! Hadn't spotted that, have removed it